Explorer/Assets/DCL/AvatarRendering/Emotes/Helpers/IEmoteStorage.cs

```diff
@@ -10,5 +11,7 @@ public interface IEmoteStorage : IAvatarElementStorage<IEmote, EmoteDTO>
         List<URN> EmbededURNs { get; }
         void AddEmbeded(URN urn, IEmote emote);
         bool TryGetLatestTransferredAt(URN nftUrn, out DateTime latestTransferredAt);
+        bool TryGetLatestOwnedNft(URN nftUrn, out NftBlockchainOperationEntry entry);
+        Dictionary<URN, Dictionary<URN, NftBlockchainOperationEntry>> AllOwnedNftRegistry { get; }
```

Comment:
Expose it as `IReadonlyDictionary` as it's not supposed to be modified from outside